pkg/common/webhook.go

```diff
@@ -27,6 +27,9 @@ const (
 	WebhookSchedulePodPath = "mutate-fluid-io-v1alpha1-schedulepod"
 
 	CertSecretName = "fluid-webhook-certs"
+
+	DatasetUseAsHCFS = "fluid.io/dataset-useas-hcfs"
+	URLPostfix       = "-ADDRESS"
```

Comment:
I suggest using underscore ( _ ) for linux environment variable names instead of (-). Because Environment variable names used by the utilities in the Shell and Utilities volume of IEEE Std 1003.1-2001 consist solely of uppercase letters, digits, and the '_' (underscore) .